cmake/sss.cmake

```diff
@@ -7,8 +7,9 @@ set(SSS_PREFIX
 )
 message(STATUS "Using sss at ${SSS_PREFIX}")
 
-set(SSS_SRC ${SSS_PREFIX}/sss.c ${SSS_PREFIX}/hazmat.c
-            ${SSS_PREFIX}/tweetnacl.c
+set(SSS_SRC
+    ${SSS_PREFIX}/sss.c ${SSS_PREFIX}/hazmat.c ${SSS_PREFIX}/tweetnacl.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/src/node/test/sss_random.cpp
```

Comment:
That does not seem right, we do not want test code in the library.

Reply:
I just moved it there because it shouldn't be part of ccfcrypto and I didn't know where else to put it. libsss expects a function with the prototype in this file (which used to be in a header). @jumaffre I think you last touched these bits - do you know where they should rather go?